src/ints/int10_memory.cpp

```diff
@@ -111,13 +111,17 @@ void INT10_ReloadFont(void) {
 		INT10_LoadFont(RealToPhysical(int10.rom.font_8_first),false,256,0,map,8);
 		break;
 	case 14:
-		if (IS_VGA_ARCH && svgaCard==SVGA_None && CurMode->mode==7) map=0x80;
-		INT10_LoadFont(RealToPhysical(int10.rom.font_14),false,256,0,map,14);
+		if (IS_VGA_ARCH && CurMode->mode == 7) {
+			map = 0x80;
+		}
+		INT10_LoadFont(RealToPhysical(int10.rom.font_14), false, 256, 0, map, 14);
 		break;
 	case 16:
 	default:
-		if (IS_VGA_ARCH && svgaCard==SVGA_None) map=0x80;
-		INT10_LoadFont(RealToPhysical(int10.rom.font_16),false,256,0,map,16);
+		if (IS_VGA_ARCH) {
+			map = 0x80;
+		}
+		INT10_LoadFont(RealToPhysical(int10.rom.font_16), false, 256, 0, map, 16);
```

Comment:
Oh, this will finally cause the correct font to be displayed on all `svga_*` and `vesa_*` machine types? If so, that makes me really happy! 😎 

(And if not, that's something to work on next 😏 )